src/ReactTerminal.js

```diff
@@ -54,46 +41,30 @@ class Terminal extends Component {
   }
 
   render() {
-    const {acceptInput, clickToFocus, theme, promptSymbol, outputRenderers, terminalId} = this.props;
-    const {emulatorState, inputStr} = this.state;
+    let {emulatorState, inputStr, ...otherProps} = this.props;
```

Comment:
Small change: can we make what's happening in these two lines of code clearer?

Could you change the two lines to:

`const {emulatorState: removedEmulatorState, inputStr: removedInputStr, ...otherProps} = this.props;`
`const {emulatorState, inputStr} = this.state`

And maybe add a small comment to the code about the use of the spread operator to remove the unwanted keys / why we're using the `emulatorState`  and `inputStr` from `state` instead of `props`.

Reply:
Good point. Done now.